pkg/userdata/helper/helper.go

```diff
@@ -157,14 +159,29 @@ NO_PROXY=%s
 no_proxy=%s`, proxy, proxy, proxy, proxy, noProxy, noProxy)
 }
 
-func SetupNodeIPEnvScript() string {
+func SetupNodeIPEnvScript(ipFamily util.IPFamily) string {
+	const defaultIfcIPv4 = `DEFAULT_IFC_IP=$(ip -o  route get 1 | grep -oP "src \K\S+")`
+
+	var defaultIfcIP string
+	switch ipFamily {
+	case util.Unspecified, util.IPv4:
```

Comment:
If you have a default case already, which should cover IPv4 why do you need this case here?

Reply:
`util.Unspecified` covers when nothing was set assuming everything will be IPv4. 

`default` covers typing mistakes `IPv5` for example.

Reply:
Ok. Will keep the default case only.